loading.bs

```diff
@@ -988,6 +1053,44 @@ A [=request=] |browserRequest| <dfn>matches the stored exchange</dfn>
     Behavior=] returning a list of lists.
 1. Return "match".
 
+<h3 algorithm id="queue-report">Queuing signed exchange report</h3>
+
+<dfn>Queuing signed exchange report</dfn> |report| is the following steps:
+
+1. Let |report body| and |policy| be the result of
+    [=generate a network error report=]</a> with |report|'s
+    [=signed exchange report/outer request=]. If the result is null, abort
+    these steps.
+
+1. If |report body|'s `"type"` is `"dns.address_changed"`, abort these steps.
```

Comment:
Should we abort these steps or send the report as-is? If aborting is right, please add a `Note:` paragraph saying why.